forc/GFS/README.md

```diff
@@ -0,0 +1,67 @@
+## Description
+
+* This package generates DATM input data for the forcing fluxes at the sea surface from the NCEP operational GFSv16 outputs.
+
+## Inputs of conversion in conv_gfs2datm_long_beta4.f
+
+* INPUT variables : from the output of GFSv16,
+    ==> longitude, latitude, time, DLWRF_surface, ULWRF_surface, DSWRF_surface, UFLX_surface, VFLX_surface, SHTFL_surface,
+      LHTFL_surface, PRATE_surface, UGRD_10maboveground, VGRD_10maboveground, PRES_surface, TMP_1hybridlevel, 
+      SPFH_1hybridlevel, UGRD_1hybridlevel, VGRD_1hybridlevel, SPFH_2maboveground, TMP_2maboveground, LAND_surface, 
+      delz, PRATE_surface, pres_hyblev1, precp, fprecp, ICEC_surface
+
+* ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.sfluxgrbf000.grib2 :: Input data are from the GFSv16 outs of initial analysis fields, 
+  except following missing variables in the initial file.
+
+./gdas.${datb}/${hhb}/atmos/gdas.t${hhb}z.sfluxgrb${hprep}.grib2 :: 6 hour forecast fields at the analysis time
+      ==> PRATE_surface, UFLX_surface, VFLX_surface
+
+./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.atmf000.nc  ==> delz
+
+## Output of conversion in conv_gfs2datm_long_beta4.f
+* OUTPUT variables for DATM
+  ==> lat, lon, time, DLWRF, ULWRF, DSWRF, vbdsf_ave, vddsf_ave, nbdsf_ave, nddsf_ave, dusfc, dvsfc, shtfl_ave, lhtfl_ave, 
+      totprcp_ave, u10m, v10m, hgt_hyblev1, psurf, tmp_hyblev1, spfh_hyblev1, ugrd_hyblev1, vgrd_hyblev1, q2m, t2m, 
+      slmsksfc, pres_hyblev1, precp, fprecp, icecsfc
```

Comment:
I think this could be better understood in the form of a table:
|Input variable | Source | Output variable|
|--| --|--|
|longitude| sfluxgrbf000.grib2| lon|
|DLWRF_surface| sfluxgrbf000.grib2|DLWRF|
|PRATE_surface| sfluxgrbf006.grib2| precp|

and so on.

Reply:
README.md has been updated.